src/app/components/modal-header.tsx

```diff
@@ -2,8 +2,7 @@ import { FiArrowLeft, FiX } from 'react-icons/fi';
 import { useNavigate } from 'react-router-dom';
 
 import { Box, IconButton } from '@stacks/ui';
```

Comment:
I wonder if it would be useful to have an `index` wrapper so we can import:
`import { Flex, styled, token } from 'leaf-styles/`

Maybe it's better to separate them though

Reply:
these files are autogenerated, not sure if we can change it, need to be checked